warre/inventory/inventory.html

```diff
@@ -23,16 +23,110 @@
 			crossorigin="anonymous"></script>
 
 	<!-- internal resource links -->
+	<!--
 	<script src="../js/global_on_page_load.js"></script>
 	<script src="../js/banner.js"></script>
 	<script src="../js/authentication.js"></script>
+	-->
+	<script src="js.cookie.js"></script>
 	<link href="inventory.css" type="text/css" rel="stylesheet">
 
 	<!-- initialize page -->
 	<script>
 		jQuery(document).ready(function(){
-			on_page_ready();
+
+		$("id-input").focus();
+
+		function selectCI() {
+			ci = true;
+			co = false;
+			stat = false;
+			if ($('#chk-out-li').hasClass('active')) {
+				$('#chk-out-li').removeClass('active');
+			}
+			if ($('#status-li').hasClass('active')) {
+				$('#status-li').removeClass('active');
+			}
+
+			if (! $('#chk-in-li').hasClass('active')) {
+				$('#chk-in-li').addClass('active');
+			}
+
+			$("id-input").focus();
+		}
+
+		var ci = true;
+		var co = false;
+		var stat = false;
+                $('#chk-in-btn').click(function () {
+			selectCI();
+		});	
+
+                $('#chk-out-btn').click(function () {
+			co = true;
+			ci = false;
+			stat = false;
+			if ($('#chk-in-li').hasClass('active')) {
+				$('#chk-in-li').removeClass('active');
+			}
+			if ($('#status-li').hasClass('active')) {
+				$('#status-li').removeClass('active');
+			}
+
+			if (! $('#chk-out-li').hasClass('active')) {
+				$('#chk-out-li').addClass('active');
+			}
+			
+			$("#id-input").focus();	
+		});	
+
+		$('#status-btn').click(function () {
+			stat = true;
+			ci = false;
+			co = false;
+			if ($('#chk-out-li').hasClass('active')) {
+				$('#chk-out-li').removeClass('active');
+			}
+			if ($('#chk-in-li').hasClass('active')) {
+				$('#chk-in-li').removeClass('active');
+			}
+		
+			if (! $('#status-li').hasClass('active')) {
+				$('#status-li').addClass('active');
+			}
+
+			$("#id-input").focus();						
 		});
+
+		$('#id-input').on('keypress', function (e) {
+         		if(e.which === 13) {
+            			$(this).attr("disabled", "disabled");
+				
+				if (!stat) {
+					Cookies.set($('#id-input').val(), ci | !co);
+					if (ci) {
+						$("#feedback").text("Checked in.");
+					} else {
+						$("#feedback").text("Checked out.");
+					}
+				} else {
+					if (Cookies.get($('#id-input').val()) == "1") {
```

Comment:
This seems a bit broken, why is this `== "1"` and not `== 1` or `== true`. You seem to be putting in boolean values above. Javascript has brain damage typing though so it might be correct.